src/stores/SearchStore.ts

```diff
@@ -688,6 +706,38 @@ export class SearchStore {
 		};
 	}
 
+	@action resetEventAutocompleteList = () => {
+		this.eventAutocompleteList = [];
+	};
+
+	private loadEventAutocompleteList = debounce((parentEventName: string) => {
+		if (this.eventAutocompleteSseChannel) {
+			this.eventAutocompleteSseChannel.close();
+			this.resetEventAutocompleteList();
```

Comment:
i think that's another bug.
this line should be outside of if statement. otherwise previos results stay in the list and the new ids append to the end of it